cg/meta/rsync.py

```diff
@@ -48,7 +49,9 @@ def run_rsync_command(self, ticket_id: int = None, dry_run: bool = False) -> Non
 
         if cases[0].data_analysis == Pipeline.SARS_COV_2:
             LOG.info("Delivering report for SARS-COV-2 analysis")
-            covid_report_path = self.covid_report_path % (str(cases[0].internal_id), ticket_id)
+            covid_report_path = glob.glob(
```

Comment:
Would be nice with type hint here

Reply:
suggest to use pathlib instead, example: https://realpython.com/python-pathlib/#counting-files